lib/gpu_cluster_status.rb

```diff
@@ -131,7 +139,7 @@ def gpus_queued_percent
       # @param job [OodCore::Job::Info]
       def queued_jobs_req_gpus_counter(job)
         if is_job_req_gpus_and_queued(job)
-         @queued_jobs_req_gpus += 1
+         @queued_jobs_requesting_gpus += 1
          @queued_gpus += job.native[:Resource_List][:nodes].slice(/gpus=(\d+)/).reverse.to_i

```

Comment:
Is `queued_gpus` and `@queued_gpus` being used anywhere? If it is no longer being used you can delete this code.